routersploit/test/test_validators.py

```diff
@@ -136,6 +136,27 @@ def test_boolify_true_7(self):
         value = "t"
         self.assertEqual(validators.boolify(value), True)
 
+    def test_choice_1(self):
+        valid_values = ["test1", "test2"]
+        selected_value = "test1"
+        self.assertEqual(validators.choice(valid_values)(selected_value), selected_value)
+
+    def test_choice_2(self):
+        valid_values = ["test1", "test2"]
+        selected_value = "t"
+
+        with self.assertRaises(OptionValidationError) as ove:
+            validators.choice(valid_values)(selected_value)
+            self.assertEqual(ove.exception, OptionValidationError)
+
+    def test_choice_3(self):
+        valid_values = ["test1", "test2"]
+        selected_value = "Test1"
+
+        with self.assertRaises(OptionValidationError) as ove:
+            validators.choice(valid_values)(selected_value)
+            self.assertEqual(ove.exception, OptionValidationError)
```

Comment:
This statement is redundant, assertRaises context manager do all the heavy lifting for us.